<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPasswordPage03</class>
 <widget class="QWidget" name="UserPasswordPage03">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目  录</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_page_pbtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>0</y>
      <width>241</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>使用者密码页</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QWidget" name="TurnOffHumidityFunction" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="turn_off_humidity_function_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关闭湿度功能</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="turn_off_humidity_function_locked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="AutomaticDefrosting" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1281</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="automatic_defrosting_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>自动除霜</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="automatic_defrosting_locked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RemoteRecording" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>1280</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="remote_recording_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>远端记录</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="remote_recording_locked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ExcessiveWarining" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>1280</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="excessive_warning_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>超标警示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="excessive_warning_locked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="WaterReplenishmentMode" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>1280</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="water_replenishment_mode_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>补水模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="water_replenishment_mode_display">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="water_replenishment_mode_text">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(0:内部水箱补水 1:外部补水)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="WaterLevelWarning" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>1280</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="water_level_warning_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>水位警示</string>
     </property>
    </widget>
    <widget class="QLabel" name="water_level_warning_display">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="water_level_warning_text">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>221</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>预计到达补水位剩余时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="water_level_warning_M">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="water_level_warning_M_sign">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>M</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TextMessaging" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>1280</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="text_messaging_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机简讯</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="text_messaging_locked">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TemperatureControl" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>1281</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="temperature_control_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度控制</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="temperature_control_checkBox_1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风口温度</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="temperature_control_checkBox_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>样品温度</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ProgramOperation" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>1280</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLabel" name="program_operation_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>程式操作</string>
     </property>
    </widget>
    <widget class="QLabel" name="program_operation_loading">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperature_control_uploading">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
